cg/services/order_validation_service/errors/case_sample_errors.py

```diff
@@ -35,6 +32,11 @@ class SampleNameRepeatedError(CaseSampleError):
     message: str = "Sample name already used"
 
 
+class SampleNameSameAsCaseNameError(CaseSampleError):
+    field: str = "name"
```

Comment:
The alphabetical order is off now

Reply:
it is super off but that is out of scope